### PR TITLE
Minor cleanup in the start guide

### PR DESCRIPTION
**Pull request by dannagle**
Fixed some of the issues I mentioned here:
https://github.com/pakyow/pakyow/issues/121

### CODE REVIEW

File: start/running.md [hunk 1]
```diff
@@ -6,8 +6,9 @@ desc: Starting the Pakyow server.
 Pakyow includes a tool for running projects locally during development. To start the
 server, run the following command in terminal from the main directory of your project:
 
-```console
-pakyow server
+```
+cd your-app-name-here
+console pakyow server
```
Comment (bryanp) [inline start/running.md]: Remove `console` here.

File: start/installing.md [hunk 2]
```diff
@@ -12,10 +12,44 @@ command in your terminal:
 bash <(curl -s https://raw.githubusercontent.com/pakyow/pakyow-deps/master/check.sh)
 ```
 
-Here are some additional links that might be helpful:
+## Install Ruby
```
Comment (bryanp) [inline start/installing.md]: Let's reorder to Mac, Linux, Windows (based on popular use).

File: start/installing.md [hunk 3]
```diff
@@ -12,10 +12,44 @@ command in your terminal:
...
+Linux users:
+  - Most Linux distros come with Ruby, but they may not be new enough.
+  - For Debian/Ubuntu/Mint:
+
+  `sudo apt-get update; sudo apt-get install ruby;`
+  - For CentOS/Red Hat/Fedora:
+
+  `sudo yum install ruby`
+
+## Install RubyGems
```
Comment (bryanp) [inline start/installing.md]: Also reorder these to Mac, Linux, Windows.

**Comment by bryanp** [comment]
Made a couple comments but looks good overall! Thanks for the contribution 😀

**Comment by dannagle** [comment]
Updated based on your comments.

**Comment by bryanp** [comment]
Looks good, thanks @dannagle!!! :heart:

**Pull request merged**